src/main/java/uk/gov/ons/ctp/response/collection/exercise/lib/common/redis/SessionResolver.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.ons.ctp.response.collection.exercise.lib.common.redis;
+
+import com.godaddy.logging.Logger;
+import com.godaddy.logging.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SessionResolver {
+
+  private static final Logger log = LoggerFactory.getLogger(SessionResolver.class);
+
+  private static final String REDIS_KEY_PREFIX = "frontstage:collection-exercise-by-survey-id:";
+
+  @Autowired private RedisUtil<Object> redisUtil;
+
+  public void removeActiveSession(String surveyId) {
```

Comment:
This function name also seems wrong.  Theres no session involved here, all this key is doing is just caching the result of an endpoint call.  If we end up having a function per key you'd want to delete, this could maybe be 'deleteFrontstageExerciseByIdKey' or something similarly descriptive.

Also a comment on this saying which endpoint call was originally called could be useful for context?  But I won't block the PR if you think it's not worth it.